SleepEngine/Engine/Utils/file_utils.cpp

```diff
@@ -15,7 +15,7 @@ std::optional<std::string> readFile(std::filesystem::path path)
 
     if (fstream.is_open())
     {
-        LOG_AND_FAIL_ERROR("file already open : {}", path);
+        LOG_AND_FAIL("file already open : {}", path);
```

Comment:
Sorry for this rename, i thought i needed to use it, renamed it and than there was no way back because of VS